test/unit/graphql_metrics_integration_test.rb

```diff
@@ -0,0 +1,997 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "test_schema"
+
+class GraphQLMetricsIntegrationTest < ActiveSupport::TestCase
```

Comment:
Commenting here, to ensure Github doesn't hide this file from reviewers by default

Reply:
it does anyway lol 😞  still need to manually expand it

Reply:
nooooooooooooooo